### ISSUE TITLE
SSLProxy with Squid

### ISSUE THREAD

**Issue by primmus** [sonertari/UTMFW#3]
Hello, I'm having a question, is it possible to use SSLProxy with Squid on linux?

because I configured sslproxy, but did not browse because the source and destination was 127.0.0.1.

Do you have any tips?

Thank you.

**Comment by sonertari** [comment]
As I guess you know by now that UTMFW uses E2Guardian+Squid in cascade. So, since the egress leg to the SSLproxy is Squid, the changes necessary to support the mode of operation required by SSLproxy is ready and available among the patches under the `ports/squid` folder (Squid should give the packets back to SSLproxy). But, since the ingress leg from the SSLproxy is E2Guardian, the changes necessary for Squid are not available (E2Guardian changes are ready of course). See the README and the presentation of SSLproxy for the details of its mode of operation.

In short, you need to review the source code of Squid and modify its connection handling code. Since each software has its own connection handling design and implementation, you cannot use the patches for E2Guardian directly, but they can give you an idea on what you need to do..
